data/cameras.xml

```diff
@@ -2066,6 +2066,46 @@
 			<Hint name="wb_offset" value="142"/>
 		</Hints>
 	</Camera>
+	<Camera make="GoPro" model="FUSION">
```

Comment:
Why is this even here?
It's DNG.

Reply:
It is based on DNG, but uses a special encoding/compression.
Anyway, it can be removed if it's not required elsewhere

Reply:
This should be reduced down to
```xml
	<Camera make="GoPro" model="FUSION" mode="dng">
		<ID make="GoPro" model="FUSION">PENTAX K-5</ID>
	</Camera>
...
```

Reply:
It will (well, should, unless they really messed with their 'DNG') work without all these `data/` changes.
I don't understand that `<ID>` tag, but that is how it is structured.
Having this minimal snipped i posted will allow for that camera to be listed in the "cameras known/supported as per the rawspeed/data/cameras.xml" lists, as seen on e.g. https://rawspeed.org/CameraSupport.html, https://www.darktable.org/resources/camera-support/, https://raw.pixls.us/?missing#repo